extensions/binding/org.eclipse.smarthome.binding.tradfri/ESH-INF/thing/thing-types.xml

```diff
@@ -69,6 +69,26 @@
             </parameter>
         </config-description>
     </thing-type>
+    <thing-type id="group">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="gateway"/>
+        </supported-bridge-type-refs>
+
+        <label>Tradfri Group</label>
+        <description>A group of Tradfri lamps</description>
+        
+        <channels>
+            <channel id="group_state" typeId="group_state"/>
```

Comment:
do groups really only have an on/off feature, or could we rather reuse the existing brightness channel here? I noticed the return value also includes the "5851" (aka DIMMER) field.